C++/common.cpp

```diff
@@ -0,0 +1,71 @@
+//#include<iostream.h>
```

Comment:
- Rename both files to better name
- Update C++ folder readme

Everything else LGTM :-)

Reply:
Updated README.md and file names.